connectd/test/run-netaddress.c

```diff
@@ -233,4 +233,5 @@ int main(int argc, char *argv[])
 	assert(IsIPv6(&wa));
 
 	common_shutdown();
+	return 0;
 }
```

Comment:
This is unnecessary in modern C.  And I guess from the commit message, this is simply a rind left over from a previous commit which added common_shutdown() and is now redundant?

Reply:
removed! (deleted whole commit, was extraneous)